src/components/action-popover/action-popover-item/action-popover-item.component.tsx

```diff
@@ -274,7 +274,16 @@ export const ActionPopoverItem = ({
   };
 
   const renderMenuItemIcon = () => {
-    return icon && <MenuItemIcon as={undefined} type={icon} />;
+    return (
+      icon && (
+        <MenuItemIcon
+          data-element="action-popover-menu-item-icon"
+          horizontalAlignment={horizontalAlignment}
+          as={undefined}
```

Comment:
I don't believe this prop does anything to the icon markup, so I would suggest we remove it.

```suggestion
```

I am assuming this was added to address some typescript errors? If so, we could work around this by following @robinzigmond's suggestion regarding prop types and introducing a default value for the `horizontalAlignment` prop.